src/plugins/appeals/unbanThread.ts

```diff
@@ -0,0 +1,35 @@
+import {useEvent} from "../../hooks";
+import {Interaction, Message} from "discord.js";
+
+useEvent("interactionCreate", async (interaction: Interaction) => {
```

Comment:
All exit points in the handler are either guard clauses or immediately after an `interaction.reply()` call that takes an identical object bar one field (`content`). As such, it would be better to pull this functionality into its own function that returns a nullable string, null meaning we don't reply with anything, with what we should reply with. This then removes the repeated `interaction.reply({ content: ..., ephemeral: true})` calls: generally repeated code should be avoided.

Reply:
Heard. Pulled the interaction.reply into its own const which should improve readability and management.